windows/MAVProxyWinBuild.bat

```diff
@@ -21,7 +21,14 @@ rem -----build the changelog-----
 "%PYTHON_LOCATION%\python" createChangelog.py
 
 rem -----Upgrade pymavlink if needed-----
-"%PYTHON_LOCATION%\Scripts\pip" install pymavlink -U
+if exist "..\..\pymavlink" (

```

Comment:
Shouldn't this be "..\..\mavlink\pymavlink"? I suppose it depends on the user configuration - whether they're using the mavlink repository or pymavlink repository.